core/src/main/java/hudson/model/StringParameterDefinition.java

```diff
@@ -69,7 +70,7 @@ public StringParameterValue getDefaultParameterValue() {
         return v;
     }
 
-    @Extension
+    @Extension @Symbol({"string","stringParam"})
```

Comment:
If the linked code were deleted, we would be unable to support

```groovy
build job: '…', parameters: [[$class: 'StringParameterValue', name: '…', value: '…']]
```

without FQNs, nor could this field be mapped to the syntax proposed in JENKINS-29922.